src/Spatial/Euclidean/Line2D.cs

```diff
@@ -27,6 +27,21 @@ public struct Line2D : IEquatable<Line2D>, IXmlSerializable
         /// </summary>
         public Point2D EndPoint { get; private set; }
 
+        /// <summary>
```

Comment:
All these parameters are redundant and are not needed in order to calculate the distance to another 2D point

Reply:
Thanks for the comment.
Some might think like you.
I'll remove these parameters.

Reply:
done in the commit https://github.com/f-frhs/mathnet-spatial/commit/847cdaf17e4784a1791d5dd11ddb8f398111cf18